packages/avatar/src/Avatar.tsx

```diff
@@ -8,7 +8,7 @@ import { useCallback, useEffect, useState } from 'react';
 import styles from './styles/Avatar.module.css';
 import { useIsMounted } from './utils';
 
-type AvatarProps = HTMLAttributes<HTMLDivElement> & {
+type AvatarProps = ComponentProps<'img'> & {
```

Comment:
This was previously typed as possible `div` attributes, but we only render a `div` if the `url` prop is omitted (or, if we attempt to load the image at the given `url` and get back a 404 or a format we don't support).

Changing the prop type to use `ComponentProps<'div'>` uncovered this, and it seems safer to just accept the intrinsic props for `img` and forward them along (knowing that some like `srcset` won't have any effect if we end up rendering a `div`).